<!-- MHonArc v2.6.16 -->
<!--X-Subject: Re: [sr #2461] Final Run error message -->
<!--X-From-R13: "Sqjneq q'Ohiretar" <rqjneqNaze&#45;erynk.pbz> -->
<!--X-Date: Mon, 14 Dec 2009 14:03:13 +0100 -->
<!--X-Message-Id: 7f080ed10912140502o5a28bef8k334b88d5351abd59@mail.gmail.com -->
<!--X-Content-Type: text/plain -->
<!--X-Reference: 20091209&#45;193526.sv0.97821@gna.org -->
<!--X-Head-End-->
<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN"
   "http://www.w3.org/TR/html4/loose.dtd">
<html>
<head>
<title>Re: [sr #2461] Final Run error message -- December 14, 2009 - 14:03</title>
<link rel="stylesheet" type="text/css" href="/mail.gna.org/archives-color-gna.css"> 
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
</head>
<body>
<!--X-Body-Begin-->
<!--X-User-Header-->
<!--X-User-Header-End-->
<!--X-TopPNI-->
<h2><img src="/mail.gna.org/images/mail.orig.png" width="48" height="48"
alt="mail" class="pageicon" />Re: [sr #2461] Final Run error message</h2>
<br />
<div class="topmenu">
<a href="../" class="tabs">Others Months</a> | <a href="index.html#00033" class="tabs">Index by Date</a> | <a href="threads.html#00033" class="tabs">Thread Index</a><br />
<span class="smaller">&gt;&gt;&nbsp;&nbsp;
[<a href="msg00032.html">Date Prev</a>] [<a href="msg00034.html">Date Next</a>] [<a href="msg00017.html">Thread Prev</a>] [<a href="msg00035.html">Thread Next</a>]
</div>

<!--X-TopPNI-End-->
<!--X-MsgBody-->
<!--X-Subject-Header-Begin-->
<h3><a name="header" href="#header">Header</a></h3>
<!--X-Subject-Header-End-->
<!--X-Head-of-Message-->
<ul class="headdata">
<li class="menuitem">
<em>Date</em>: Mon, 14 Dec 2009 14:02:32 +0100</li>
<li class="menuitem">
<em>Cc</em>: mikaela.stewart@xxxxxxxxx, relax-devel@xxxxxxx</li>
<li class="menuitem">
<em>Message-id</em>: &lt;<a href="msg00033.html">7f080ed10912140502o5a28bef8k334b88d5351abd59@mail.gmail.com</a>&gt;</li>
<li class="menuitem">
<em>References</em>: &lt;20091209-193526.sv0.97821@xxxxxxx&gt;</li>
</ul>
<!--X-Head-of-Message-End-->
<!--X-Head-Body-Sep-Begin-->
</div><!-- end headdata -->
<br />
<h3><a name="content" href="#content">Content</a></h3>
<div class="postedby">Posted by <strong>Edward d'Auvergne</strong> on December 14, 2009 - 14:03:</div>
<div class="msgdata">
<!--X-Head-Body-Sep-End-->
<!--X-Body-of-Message-->
<pre style="margin: 0em;">Dear Mikaela,

Thank you for you report.  Please note though that the bug tracker
(<a  rel="nofollow" href="https://web.archive.org/web/https://gna.org/bugs/?group=relax">https://gna.org/bugs/?group=relax</a>) is a better place to report bugs
than the support request tracker.  Support requests are for new
features, crazy wishes, etc.  To see the previously submitted bugs,
modify the 'Display criteria' at the top.  This is a bug that has been
reported before
(<a  rel="nofollow" href="/mail.gna.org/public/relax-users/2009-11/msg00000.html">https://mail.gna.org/public/relax-users/2009-11/msg00000.html</a>).
Unfortunately Jolyon did not submit a bug report so I was unable to
track down the issue
(<a  rel="nofollow" href="/mail.gna.org/public/relax-users/2009-11/msg00001.html">https://mail.gna.org/public/relax-users/2009-11/msg00001.html</a>).  If
you could submit a report, that would be much appreciated!  If you
could attach the script with a tiny amount of data (i.e. a small
archive file I can untar and run 'relax script.py' and reproduce the
error) will allow me to fix it within a few minutes (assuming I'm not
on holidays).

Another test would be if you have access to the subversion program and
downloaded the newest 1.3 line version of relax (instructions at
<a  rel="nofollow" href="/mail.gna.org/public/relax-users/2008-09/msg00002.html">https://mail.gna.org/public/relax-users/2008-09/msg00002.html</a>).  Maybe
the bug has already accidentally been fixed?

Cheers,

Edward


2009/12/9 anonymous &lt;NO-REPLY.INVALID-ADDRESS@xxxxxxx&gt;:
</pre><blockquote class="blockquote"><pre style="margin: 0em;">

URL:
 &lt;<a  rel="nofollow" href="http://gna.org/support/?2461">http://gna.org/support/?2461</a>&gt;

                Summary: Final Run error message
                Project: relax
           Submitted by: None
           Submitted on: Wednesday 12/09/2009 at 19:35 CET
               Category: None
               Priority: 5 - Normal
               Severity: 3 - Normal
                 Status: None
                Privacy: Public
            Assigned to: None
       Originator Email: mikaela.stewart@xxxxxxxxx
            Open/Closed: Open
        Discussion Lock: Any
       Operating System: GNU/Linux

   _______________________________________________________

Details:

I am running relax using the full_analysis.py script and after running all 
of
the models until they converged I tried to run &quot;final&quot; to pick the best 
model
and do monte-carlo simulations and I get the following traceback...

relax&gt; monte_carlo.create_data(method='back_calc')
Traceback (most recent call last):
 File &quot;/usr/local/bin/relax&quot;, line 418, in ?
   Relax()
 File &quot;/usr/local/bin/relax&quot;, line 127, in __init__
   self.interpreter.run(self.script_file)
 File &quot;/usr/local/relax-1.3.3/relax-1.3/prompt/interpreter.py&quot;, line 273, in
run
 File &quot;/usr/local/relax-1.3.3/relax-1.3/prompt/interpreter.py&quot;, line 534, in
run_script
 File &quot;/usr/local/relax-1.3.3/relax-1.3/prompt/interpreter.py&quot;, line 430, in
interact_script
 File &quot;full_analysis.py&quot;, line 668, in ?
   Main(self.relax)
 File &quot;full_analysis.py&quot;, line 368, in __init__
   monte_carlo.create_data()
 File &quot;/usr/local/relax-1.3.3/relax-1.3/prompt/monte_carlo.py&quot;, line 91, in
create_data
 File &quot;/usr/local/relax-1.3.3/relax-1.3/generic_fns/monte_carlo.py&quot;, line
76, in create_data
 File &quot;/usr/local/relax-1.3.3/relax-1.3/specific_fns/model_free/main.py&quot;,
line 508, in create_mc_data
 File
&quot;/usr/local/relax-1.3.3/relax-1.3/specific_fns/model_free/mf_minimise.py&quot;,
line 61, in back_calc
 File
&quot;/usr/local/relax-1.3.3/relax-1.3/specific_fns/model_free/mf_minimise.py&quot;,
line 773, in minimise
AttributeError: 'SpinContainer' object has no attribute 'model'


I am also not getting errors with any of my local parameters (S2, te, etc).
I thought maybe they weren't calculated until the final run, but now that 
I'm
having problems with the final run maybe this is why?

Thank you very much for your help!




   _______________________________________________________

Reply to this item at:

 &lt;<a  rel="nofollow" href="http://gna.org/support/?2461">http://gna.org/support/?2461</a>&gt;

_______________________________________________
 Message sent via/by Gna!
 <a  rel="nofollow" href="http://gna.org/">http://gna.org/</a>


_______________________________________________
relax (<a  rel="nofollow" href="http://nmr-relax.com">http://nmr-relax.com</a>)

This is the relax-devel mailing list
relax-devel@xxxxxxx

To unsubscribe from this list, get a password
reminder, or change your subscription options,
visit the list information page at
<a  rel="nofollow" href="/mail.gna.org/listinfo/relax-devel">https://mail.gna.org/listinfo/relax-devel</a>

</pre></blockquote><pre style="margin: 0em;">


</pre>
<!--X-Body-of-Message-End-->
<!--X-MsgBody-End-->
<!--X-Follow-Ups-->
</div><!-- end msgdata -->
<br />
<h3><a name="related" href="#related">Related Messages</a></h3>
<div class="relateddata">
<!--X-Follow-Ups-End-->
<!--X-References-->
<ul><li><strong>References</strong>:
<ul>
<li><strong><a name="00017" href="msg00017.html">[sr #2461] Final Run error message</a></strong>
<ul><li><em>From:</em> anonymous</li></ul></li>
</ul></li></ul>
<!--X-References-End-->
<!--X-BotPNI-->
</div><!-- end relateddata -->
<!-- NoBotLinksApartFromRelatedMessages -->

<!--X-BotPNI-End-->
<!--X-User-Footer-->
<!--X-User-Footer-End-->
<div class="footer"></div><br />
<div class="right">Powered by <a href="http://www.mhonarc.org">MHonArc</a>, Updated Mon Dec 14 16:00:18 2009</div>  
</body>
</html>
